### ISSUE TITLE
UX: "count me in" should be "create account"

### ISSUE THREAD

**Issue by milahu** [syphon-org/syphon#714]

**Comment by ereio** [comment]
There are many times where Syphon chooses to be more playful with language when in English. 

I'd prefer it remain "count me in" when in English, but perhaps we could mark the button as `create-account-playful` or `create-account-informal` so other translations could identify something similar to maintain the cheerful / playful language Syphon aims to convey in every language. This is something Discord and other modern apps do much more extensively than Syphon already.

Let me know your thoughts on this with this context and thanks for the feedback!

**Comment by 0x1a8510f2** [comment]
While personally I don't really mind either way, perhaps "sign me up!" (exclamation mark optional) could be considered a good middle ground? It keeps the playful tone but has more meaning when read literally.

**Comment by milahu** [comment]
"count me in" is not intuitive, because the meaning depends on context

\- wanna sell your soul?
\- count me in!

a button saying "sell my soul" is intuitive

**Comment by ereio** [comment]
> While personally I don't really mind either way, perhaps "sign me up!" (exclamation mark optional) could be considered a good middle ground? It keeps the playful tone but has more meaning when read literally.

makes sense, happy to change it to `sign me up` for now if that would help with clarity and is what you're going for here @milahu

Just for more context, Windows uses `got it` a lot and I think I actually got `count me in` from a system confirmation from Windows iirc
